crt_portal/cts_forms/templates/forms/complaint_view/show/correspondent_info.html

```diff
@@ -26,7 +26,18 @@
       </tr>
       <tr>
         <th>Email</th>
-        <td>{{ data.contact_email|default:"—" }}</td>
+        <td>
+          <div class="display-flex">
+            <span>{{ data.contact_email|default:"—" }}</span>
+            {% if data.related_reports and data.related_reports|length > 1 %}
+              <span class="margin-left-auto">
+                <a href="{% url 'crt_forms:crt-forms-index' %}?contact_email={{data.contact_email}}">View All ({{data.related_reports|length}})</a>
```

Comment:
Wondering if we should list all sections explicitly in the query parameters, otherwise email matches from other sections may be hidden due to how we automagically filter for the user's given section in the view all page.

Reply:
I realize what I said might not be clear, so here's a screenshot version:

Suppose I am an user with ELS, who sees this nifty feature. I see this report has one other email associated with it so I click on this nifty new feature:

![2020-12-28_10-35](https://user-images.githubusercontent.com/3013175/103236066-9ee79b00-48f8-11eb-888f-349c53f34123.png)

But much to my surprise, only one shows up:

![2020-12-28_10-35_1](https://user-images.githubusercontent.com/3013175/103236088-b030a780-48f8-11eb-964b-e04b93fb394d.png)

The "right" solution ime would be to _not_ filter for ELS anymore and then both reports would show up as advertised:

![2020-12-28_10-34](https://user-images.githubusercontent.com/3013175/103236101-bcb50000-48f8-11eb-8bb7-91cc54d6c0d6.png)

Reply:
Great question, bringing back to the group for discussion!!!

Our current implementation doesn't appear to let us conveniently override the "global section filter" 😅

Reply:
Revised this a bit to better handle this situation and reveal reports from all sections when following that link. Thanks @cantsin!